features/machine/alerting.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Alerting for machine-api
+
+  # @author jhou@redhat.com
+  @admin
+  @destructive
+  Scenario Outline: Alert should be fired when operator is down
+    Given I switch to cluster admin pseudo user
+
+    # scale down cvo and operators
+    When I run the :scale admin command with:
+      | resource | deployment                |
+      | name     | cluster-version-operator  |
+      | replicas | 0                         |
+      | n        | openshift-cluster-version |
+    Then the step should succeed
+    And I register clean-up steps:
+    """
+    When I run the :scale admin command with:
+      | resource | deployment                |
+      | name     | cluster-version-operator  |
+      | replicas | 1                         |
+      | n        | openshift-cluster-version |
+    Then the step should succeed
+    """
+    When I run the :scale admin command with:
+      | resource | deployment            |
+      | name     | <operator>            |
+      | replicas | 0                     |
+      | n        | openshift-machine-api |
+    Then the step should succeed
+    And I register clean-up steps:
+    """
+    When I run the :scale admin command with:
+      | resource | deployment            |
+      | name     | <operator>            |
+      | replicas | 1                     |
+      | n        | openshift-machine-api |
+    Then the step should succeed
+    """
+
+    And I use the "openshift-monitoring" project
+    And evaluation of `secret(service_account('prometheus-k8s').get_secret_names.find {|s| s.match('token')}).token` is stored in the :token clipboard
+
+    # allow time for the alert to fire
+    Given 180 seconds have passed
```

Comment:
How about use wait up to instead of hard waiting ?

Reply:
Good suggestion, I updated it.